lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1050,13 +1085,21 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	// a deadline is requested.
 	deadline := inp.Params.DeadlineHeight
 
-	// Set the deadline if target conf is specified.
+	// Set the deadline if it was specified.
 	//
 	// TODO(yy): upgrade `falafel` so we can make this field optional. Atm
 	// we cannot distinguish between user's not setting the field and
 	// setting it to 0.
-	if in.TargetConf != 0 {
-		deadline = fn.Some(int32(in.TargetConf) + currentHeight)
+	if in.DeadlineDelta != 0 {
+		deadline = fn.Some(int32(in.DeadlineDelta) + currentHeight)
+	}
+
+	startingFeeRate := inp.Params.StartingFeeRate
+
+	// We only set the starting fee rate if it was specified else we keep
+	// the existing one.
+	if feerate.IsSome() {
+		startingFeeRate = feerate
```

Comment:
So for the use case we have in mind, we never set a deadline nor budget, but only set the starting fee rate of the fee function? Is that enough to have it _not_ auto bump?

Reply:
yes for now, but we still need to add the external/internal feature in LND 19. But it fixes the case where the user tried to bump the feeRate of the CPFP. Now the user selects conf_target=1, which will translate to the starting feeRate of the fee func., the default fee func. width is 1008 for CPFP so the full budget will never be reached.